src/components/application_manager/src/hmi_capabilities_impl.cc

```diff
@@ -1966,6 +2022,11 @@ void HMICapabilitiesImpl::set_ccpu_version(const std::string& ccpu_version) {
   ccpu_version_ = ccpu_version;
 }
 
+bool HMICapabilitiesImpl::matches_ccpu_version(
```

Comment:
ALso should be Camel case.

Reply:
[done](https://github.com/smartdevicelink/sdl_core/pull/3269/files#diff-f67d217da60dca297da36f9d5dcbeac3R1634)